mobile/src/bundles/users/components/new-account-header/new-account-header.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import { Text, View } from '~/bundles/common/components/components';
+import {
+    OnboardingScreenName,
+    TextCategory,
+} from '~/bundles/common/enums/enums';
+import { globalStyles } from '~/bundles/common/styles/styles';
+
+import { StepProgressBar } from './step-progress-bar';
+import { StepTrack } from './step-track';
+
+type Properties = {
+    title: (typeof OnboardingScreenName)[keyof typeof OnboardingScreenName];
```

Comment:
Can you use ValueOf in this case ?

Reply:
This should be written to this [PR](https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/pull/119). Because I had some issues with github and Tetian's code was showing here

Reply:
Yes, that is my code. I fixed it, thank you for the advice.